src/pages/LogInWithShortLivedTokenPage.js

```diff
@@ -57,19 +57,20 @@ class LogInWithShortLivedTokenPage extends Component {
         const email = lodashGet(this.props.route.params, 'email', '');
         const shortLivedToken = lodashGet(this.props.route.params, 'shortLivedToken', '');
 
-        // User is trying to transition with a different account than the one they are currently signed in as so we will sign out and then sign in with the new authToken
-        if (email !== this.props.session.email) {
-            Session.signOut();
+        const signedIn = this.props.session && this.props.session.authToken;
+        if (!signedIn) {
```

Comment:
```suggestion
        const isUserSignedIn = this.props.session && this.props.session.authToken;
        if (!isUserSignedIn) {
```
to keep variable naming consistent with what is written here https://github.com/Expensify/App/blob/main/STYLE.md#boolean-variables-and-props

Reply:
Ah thanks for pointing that out!